iceoryx_posh/include/iceoryx_posh/iceoryx_posh_types.hpp

```diff
@@ -133,9 +133,8 @@ constexpr uint32_t MAX_PROCESS_NUMBER = 300U;
 /// Maximum number of services, which can be found.
 /// This limitation is coming due to the fixed capacity of the cxx::vector (This doesn't limit the offered number of
 /// instances)
-/// @todo #415 increase number back to 50 once service registry is available via shared memory
 /// @todo #1074 define and set the limits concerning the service discovery
-constexpr uint32_t MAX_NUMBER_OF_SERVICES = 10U;
+constexpr uint32_t MAX_NUMBER_OF_SERVICES = 100U;
```

Comment:
Most probably we want something like `MAX_PUBLISHER + MAX_SERVER`. Maybe even `MAX_PUBLISHER * 0.8 + MAX_SERVER` or something like this if we assume there will be multi-publisher which share the same `ServiceDiscovery`. But that can be decided later on.

Reply:
Yes we have issue https://github.com/eclipse-iceoryx/iceoryx/issues/1074 for this.